django_netjsonconfig/tests/test_admin.py

```diff
@@ -255,6 +256,22 @@ def test_preview_template(self):
         self.assertNotContains(response, 'system')
         self.assertNotContains(response, 'hostname')
 
+    def test_wrong_uuid_field_in_change_404(self):
+        """
+            Device not saved in the databse
+        """
+        d = Device(name='test')
+        path = reverse('admin:django_netjsonconfig_device_change', args=[d.pk])
+        """
+            If views are rendered, a does not exist exception shall be raised
+            leading to internal server error Http response 500.
+        """
+        try:
+            response = self.client.get(path)
+        except Device.DoesNotExist:
+            response = HttpResponse(status=500)
```

Comment:
this should be removed, only the line `response = self.client.get(path)` should be kept.
If the exception is raised, the test will fail with an error and that's fine.